docs/workshop-schedule.md

```diff
@@ -3,18 +3,18 @@ title: Workshop Schedule
 nav_title: Schedule
 ---
 
-| Time | Content |
+| Time | Topic | 
 |------|---------|
-| 9:00 - 10:15 am  | Welcome and introduction | 
-| 10:15 - 10:45 am | Organizing your projects and files |
+| 9:00 - 10:15 am  | [Welcome](https://docs.google.com/presentation/d/1GgqWl6las6ekQtKLmPzJ8UD_cftIbJjDTS2gJpStcMQ/edit?usp=sharing) <br> [Why does reproducibility matter?](https://docs.google.com/presentation/d/1qfulAR4jD0KS7NfrLHpwT6SWl-7APBmqNnAwGpXX5oo/edit?usp=sharing) |
+| 10:15 - 10:45 am | [Project Organization](https://docs.google.com/presentation/d/1ncqxXlC0-PGEK-yE7S-nDYnMPhrOUPbI95EJy283wCs/edit?usp=sharing) |
 | 10:45 - 11:00 am | Coffee break | 
-| 11:00 - 11:45 am | Introduction to UNIX and the command line |
-| 11:45 - 12:30 pm | Introduction to git <br> Forking and cloning a repository |
-| 12:30 - 1:15 pm  | Lunch |
-| 1:15 - 2:00 pm   | Shell scripting |
-| 2:00 - 2:30 pm   | The "stage/commit/push" git cycle <br> `.gitignore` files |
-| 2:30 - 2:45 pm   | Coffee break |
-| 2:45 - 3:15 pm   | Scripts, notebooks, and literate programming |
-| 3:15 - 3:45 pm   | Managing package versions with `renv` <br> Working with branches in git |
-| 3:45 - 4:15 pm   | Documenting your projects |
+| 11:00 - 11:30 am | [Introduction to UNIX and the command line](https://docs.google.com/presentation/d/1WPXkItJZEUXMY20cLrdMXHiBC2PyunR14RUSDg4nfIc/edit?usp=sharing) <br> [Resource: UNIX Reference](./unix_reference.html) <br> [Resource: UNIX Cheatsheet](./unix_quick_reference.pdf) |
+| 11:30 - 12:00 pm | [Introduction to Git; Forking and cloning a repository](https://docs.google.com/presentation/d/1eiGZA4PYBKJx5HDCo3UDOAB7q415gg96TehxilCHwlA/edit?usp=sharing) |
+| 12:00 - 1:00 pm  | Lunch |
+| 1:00 - 1:45 pm   | [Shell scripting](https://docs.google.com/presentation/d/1SDUyYVNgvDDRodVqmDQPVQ5wnjQesWfBTg0EAcdbcSo/edit?usp=sharing) |
+| 1:45 - 2:15 pm   | [The Git Stage/Commit/Push workflow and `.gitignore` files](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing) |
```

Comment:
Just noting this deck had permission that allowed comments; I updated it to view only.

Reply:
!! good catch.